content/components/inline-message.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Inline Message
+description: InlineMessage is used to inform the user about the result of an action within the content.
```

Comment:
We use sentence-case for component names every where else.

For example:

- `"InlineMessage"` => `"inline message"`
- `"NavList"` => `"nav list"`

---

I personally prefer PascalCase, but I think it's better to be consistent.